src/test/java/hudson/plugins/build_timeout/operations/AbortAndRestartOperationTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * The MIT License
+ * 
+ * Copyright (c) 2015 Jochen A. Fuerbacher
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package hudson.plugins.build_timeout.operations;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.LinkedList;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.SleepBuilder;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import hudson.model.Cause;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.plugins.build_timeout.*;
+import hudson.plugins.build_timeout.BuildTimeOutJenkinsRule;
+import hudson.plugins.build_timeout.BuildTimeoutWrapper;
+import hudson.plugins.build_timeout.impl.AbsoluteTimeOutStrategy;
+
+
+public class AbortAndRestartOperationTest {
+    
+    @Rule
+    public BuildTimeOutJenkinsRule j = new BuildTimeOutJenkinsRule();
+
+    @Test
+    public void testAbortAndRestartOnce() throws Exception {
+        
+        FreeStyleProject testproject = j.createFreeStyleProject();
+
+        QuickBuildTimeOutStrategy strategy = new QuickBuildTimeOutStrategy(5000);
+        AbortAndRestartOperation operation = new AbortAndRestartOperation(1); //Number of restarts
+        LinkedList<BuildTimeOutOperation> list = new LinkedList<BuildTimeOutOperation>();
+        list.add(operation);
```

Comment:
JFYI: [`Arrays.asList`](http://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#asList(T...)) is useful for this purpose.